app/2.0/docs/tools/glossary.md

```diff
@@ -0,0 +1,160 @@
+
+## dependency variants
+Polymer dependencies vary based on the version of Polymer in which you are implementing your element. In the context of Polymer tools, a variant is a version of your element that is built using a specific version of Polymer. 
+
+Variants are useful for testing elements against different versions of Polymer, where [`bower.json`](#bower-json) would normally only let you install and test against one version
+
+If you run `polymer install --variants`, the `polymer install` tool creates separate builds for your element using the versions of Polymer you specify. You can use the [Polymer development server](#polyserve) to test these variants. 
+
+## dynamic dependency
+A dynamic dependency is a dependency that is imported at runtime, using a mechanism such as `importHref`. Compare [static dependency](#static-dependency).
+
+# E
+## eager loading
+Eager loading refers to the process of loading dependencies statically, rather than on demand. Compare [lazy loading](#lazy-loading).
+
+## entry point
+The HTML actually loaded by the browser to start the application. Usually `index.html`. In an application built with an [app shell](#app-shell) architecture, the entrypoint is loaded for many different URLs.
+
+# F
+## fragment
+Fragments are used to define bundles and are often associated with routes or non-critical resources. A fragment is a lazily-loaded HTML file. 
+
+# G
+## git
+A distributed version control system. bower depends on git to install components from their GitHub repositories.
+
+## gulp
+A streamed JavaScript task runner that can be used to set up a custom build process using the [`polymer-build` library](#polymer-build).
+
+# H
+## hydrolysis
+This tool has been renamed to [Polymer Analyzer](#polymer-analyzer). It is a static analysis framework for Web Components. 
+
+# L
+## lazy loading
+Lazy loading takes advantage of routing to load elements only when they are required for render. Used correctly, lazy loading can improve performance.
+
+For an example implementation of lazily-loaded elements, see the [Shop app](https://github.com/Polymer/shop). In the Shop app, views are elements which are instantiated only when they are routed to. 
+
+## linter
+A linter is a piece of software that analyzes code for suspicious-looking constructions that can potentially cause errors; for example, for example, registering an element without specifying its tag name, or using invalid data-binding syntax. The Polymer tools include [`polymer-linter`](#polymer-linter), which does this for Polymer apps and elements.
+
+# M
+## monolithic
+A monolithic app is one that loads all of its code statically and does not use an app shell architecture or dynamic code loading.
+
+# N
+## Node.js
+An event-driven JavaScript runtime environment. The [Polymer CLI](#polymer-cli) and [bower](#bower) are both built on Node.
+
+## npm
+Package management system for [Node.js](#node-js).
```

Comment:
I think it is okay to have them. I have linked out to nodejs.org for consistency with other entries (e.g the entry on Bower links out to bower.io).